docs/dictionary/command/play.lcdoc

```diff
@@ -102,6 +104,12 @@ by using the put <command> :
 > and <videoClip> for details. To play a wider variety of formats, 
 > use a <player> object.
 
+Changes:
```

Comment:
@livecodesam I suggest replacing the whole paragraph with:

The use of <QuickTime> was deprecated in version 8.1 of LiveCode. Thus, the `play videoClip ..` form of the `play` command is no longer supported, since it was relying on <QuickTime>.